packages/components/src/components/hds/app-side-nav/index.ts

```diff
@@ -39,6 +38,7 @@ export default class HdsAppSideNav extends Component<HdsAppSideNavSignature> {
 
   constructor(owner: unknown, args: HdsAppSideNavSignature['Args']) {
     super(owner, args);
+    this.isMinimized = this.args.isMinimized ?? false; // sets the default state on 'desktop' viewports
```

Comment:
[question] I infer this has not  been converted to a getter because we don't want consumers to update dynamically its value, once the component is rendered; right?

Reply:
It seems like it should only be set once from the args, if passed, then updated via the actions.

Reply:
If it used a getter instead then it would also need a setter. (I was getting an error when I initially changed it to use a getter so decided to just set it in the constructor instead.)

Let me know if you think it would be better to use a getter & setter though.

Reply:
No, makes sense. Thanks for the clarification.